database/src/migrations/release.0.8.0/biohub.sql

```diff
@@ -968,19 +995,19 @@ COMMENT ON TABLE submission_observation IS 'A listing of historical data submiss
 --
 
 CREATE TABLE submission_spatial_component(
-    submission_spatial_component_id    integer                     GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
-    submission_observation_id          integer                     NOT NULL,
-    spatial_component                  jsonb                       NOT NULL,
-    geometry                           geometry(geometry, 3005),
-    geography                          geography(geometry),
+    submission_spatial_component_id    integer           GENERATED ALWAYS AS IDENTITY (START WITH 1 INCREMENT BY 1),
+    submission_observation_id          integer           NOT NULL,
+    spatial_component                  jsonb             NOT NULL,
+    geometry                           point,
+    geography                          point,
     secured_spatial_component          jsonb,
-    secured_geometry                   geometry(geometry, 3005),
-    secured_geography                  geography(geometry),
-    create_date                        timestamptz(6)              DEFAULT now() NOT NULL,
-    create_user                        integer                     NOT NULL,
+    secured_geometry                   point,
+    secured_geography                  point,

```

Comment:
Does the point type work the same with the PostGIS functions? Doing spatial intersect queries, etc?

Reply:
For example: `spatial-repository.ts -> _whereBoundaryIntersects`.

Reply:
No, of course not! having logical to physical mapping issues with the tool. Nice catch! please find corrected.